### PR TITLE
add host to logformat to support logging it in web worker logs

### PR DESCRIPTION
**Pull request by PankhudiB**
Add URL host to the transport web worker logs #165

Modified the tools.LogFormatter to incorporate & log hostname of request

### CODE REVIEW

**Comment by PankhudiB** [comment]
@prestonvasquez Can you please review the PR ?

File: internal/transport/transport.go [hunk 1]
```diff
@@ -453,6 +453,7 @@ func webWorker(ctx context.Context, workerID int, jobs <-chan *webJob) {
 			WorkerID:   workerID,
 			WorkerName: "web",
 			Duration:   time.Since(start),
+			Host:       rsp.Request.URL.Host,
```
Comment (prestonvasquez) [inline internal/transport/transport.go:456]: We should escape the host before logging it. See [here](https://github.com/alpine-hodler/gidari/blob/main/internal/transport/transport.go#L449-L450) on how to do that.

File: tools/log_formatter_test.go [hunk 2]
```diff
@@ -42,6 +42,13 @@ func TestLogFormatter(t *testing.T) {
 			t.Errorf("expected '{d:1s}', got '%s'", lf.String())
 		}
 	})
+	t.Run("host", func(t *testing.T) {
```
Comment (prestonvasquez) [inline tools/log_formatter_test.go:45]: Thank you for writing this test! Can we also include `Host` in the "all" test [here](https://github.com/alpine-hodler/gidari/blob/main/tools/log_formatter_test.go#L68) ?

**Review by prestonvasquez — CHANGES_REQUESTED** [review]

File: tools/log_formatter_test.go (hunk 2 above)
Comment (PankhudiB) [reply tools/log_formatter_test.go:45]: Oh, missed that test..will do both the changes. Thankyou

File: tools/log_formatter_test.go [hunk 3]
```diff
@@ -65,9 +72,9 @@ func TestLogFormatter(t *testing.T) {
 	})
 	t.Run("all", func(t *testing.T) {
 		t.Parallel()
-		lf := LogFormatter{WorkerID: 1, WorkerName: "worker", Duration: time.Second, Msg: "hello", UpsertedCount: 1}
-		if lf.String() != "{w:1, worker:worker, d:1s, u:1, m:hello}" {
-			t.Errorf("expected '{w:1, worker:worker, d:1s, u:1, m:hello}', got '%s'", lf.String())
+		lf := LogFormatter{WorkerID: 1, WorkerName: "worker", Duration: time.Second, Host: "localhost", Msg: "hello", UpsertedCount: 1}
```
Comment (prestonvasquez) [inline tools/log_formatter_test.go:75]: @PankhudiB There is a minor linting issue here.

**Review by prestonvasquez — CHANGES_REQUESTED** [review]

**Comment by prestonvasquez** [comment]
Thank you for contributing @PankhudiB !

**Pull request merged**